queue/private.php

```diff
@@ -1,88 +1,83 @@
 <?php $root = realpath($_SERVER["DOCUMENT_ROOT"]) ?>
 <!DOCTYPE html>
-<html class='no-js theme-light' lang="">
+<html class="no-js theme-light" lang="" ng-app="queueApp">
 <head>
 	<title>Queue Test</title>
 	<?php require_once "$root/server/header.php" ?>
-	<link rel='stylesheet' href='/queue/queue_private.css'>
-	<meta http-equiv='refresh' content='600'/>
+	<link rel="stylesheet" href="/queue/queue.css">
+	<link rel="stylesheet" href="/queue/queue_private.css">
+	<?php require_once "$root/server/angularjs_dependencies.php" ?>
+	<script src="/queue/queue.js"></script>
+	<script src="/queue/queue_service.js"></script>
+	<script src="/queue/queue_private.js"></script>
```

Comment:
Is this the way you got around the flickering template? By putting all the scripts at the top?

Reply:
Nope. That was a temporary fix until I got ng-cloak working.

[stackoverflow about this](https://stackoverflow.com/questions/15538125/angularjs-in-head-vs-body)

I'll fix this real quick.